src/Simulator/IncrementalRedstoneSimulator/RedstoneComparatorHandler.h

```diff
@@ -54,7 +54,13 @@ class cRedstoneComparatorHandler : public cRedstoneHandler
 
 			virtual bool Item(cBlockEntity * a_BlockEntity) override
 			{
-				auto & Contents = static_cast<cBlockEntityWithItems *>(a_BlockEntity)->GetContents();
+				auto BlockEntityWithItems = dynamic_cast<cBlockEntityWithItems *>(a_BlockEntity);
+				if (!BlockEntityWithItems)
```

Comment:
Looks good to me but could you explain the falsy/truthyness of `BlockEntityWithItems` to me?